packages/xo-web/src/xo-app/backup-ng/new/_RemoteProxyWarning.js

```diff
@@ -0,0 +1,44 @@
+import _ from 'intl'
+import addSubscriptions from 'add-subscriptions'
+import decorate from 'apply-decorators'
+import Icon from 'icon'
+import PropTypes from 'prop-types'
+import React from 'react'
+import Tooltip from 'tooltip'
+import { injectState, provideState } from 'reaclette'
+import { keyBy } from 'lodash'
+import { subscribeRemotes } from 'xo'
+
+const RemoteProxyWarning = decorate([
+  addSubscriptions({
+    remotes: cb =>
+      subscribeRemotes(remotes => {
+        cb(keyBy(remotes, 'id'))
+      }),
+  }),
+  provideState({
+    computed: {
+      showWarning: (_, { id, proxyId, remotes = {} }) => {
+        const remote = remotes[id]
+        if (proxyId === null) {
```

Comment:
With this `proxyId: PropTypes.string`, `proxyId` can be `null` ?

Reply:
Yes, because is not required.

Reply:
I don't see why it's weird!

What i want is to show warning when the remote is not linked to the selected proxy. 

But, the UI values is not the same with the server values, so, i convert the UI value to the server value to  compare.